cranelift-codegen/meta/src/cdsl/instructions.rs

```diff
@@ -1296,15 +1296,15 @@ impl Into<InstSpec> for BoundInstruction {
 mod test {
     use super::*;
     use crate::cdsl::formats::InstructionFormatBuilder;
-    use crate::cdsl::operands::{OperandKindBuilder, OperandKindFields};
+    use crate::cdsl::operands::{OperandKind, OperandKindFields};
     use crate::cdsl::typevar::TypeSetBuilder;
     use crate::shared::types::Int::{I32, I64};
 
     fn field_to_operand(index: usize, field: OperandKindFields) -> Operand {
         // Pretend the index string is &'static.
         let name = Box::leak(index.to_string().into_boxed_str());
         // Format's name / rust_type don't matter here.
-        let kind = OperandKindBuilder::new(name, name, field).build();
+        let kind = OperandKind::new(name, name, field);
```

Comment:
I don't understand the comment on 1306: why can we do `name, name` here and it doesn't matter?